pandas/tests/io/json/test_pandas.py

```diff
@@ -650,6 +650,21 @@ def test_series_roundtrip_timeseries(self, orient, numpy, datetime_series):
 
         tm.assert_series_equal(result, expected)
 
+    def test_series_roundtrip_periodseries(self, orient, period_series):
+        # GH32665: Fix to_json when converting Period column/series
+        data = period_series.to_json(orient=orient)
+        result = pd.read_json(data, typ="series", orient=orient)
```

Comment:
This issue affects reading to a DataFrame as well right? If so can you test that?

Reply:
OK, I can do that.